storage/src/cache/fscache/mod.rs

```diff
@@ -290,7 +292,9 @@ impl FileCacheEntry {
             is_zran,
             dio_enabled: true,
             need_validation,
-            batch_size: RAFS_DEFAULT_CHUNK_SIZE,
+            // If none, use default 0 since it's not used.
+            // e.g., at build time.
+            user_io_batch_size: mgr.user_io_batch_size.unwrap_or(0),
```

Comment:
Can the user set it to 0? It seems we have a `default_user_io_batch_size`.

Reply:
It is not wrapped by Option anymore.
Users can set it to zero if they set it in the config file.